package.json

```diff
@@ -98,6 +98,7 @@
     "react-native-vector-icons": "^6.6.0",
     "react-native-web": "^0.11.4",
     "react-native-web-webview": "^0.2.8",
+    "react-native-wkwebview-reborn": "^2.0.0",
```

Comment:
what is this?

Reply:
@sirpy I pushed it by mistake. Removed in new commit.